ext/session/session.c

```diff
@@ -2092,6 +2092,11 @@ PHP_FUNCTION(session_set_save_handler)
 		RETURN_TRUE;
 	}
 
+	zend_error(E_DEPRECATED, "Calling session_set_save_handler() with more than 2 arguments is deprecated");
+	if (UNEXPECTED(EG(exception))) {
+		RETURN_THROWS();
+	}
```

Comment:
This should be _after_ the ZPP check, because a call to 10 or more arguments is going to cause a ArgumentCountException.

Reply:
ACK